openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py

```diff
@@ -456,46 +440,43 @@ def test_prerequisites(mock_prereq_course_scores, user_application):
     assert expected_result == actual_result
 
 
-@pytest.mark.parametrize('resume, cover_letter_file, cover_letter_text, expected_fieldsets', [
-    (TEST_RESUME, None, None, ALL_FIELDSETS),
-    (None, TEST_COVER_LETTER_FILE, None, ALL_FIELDSETS),
-    (None, None, TEST_COVER_LETTER_TEXT, ALL_FIELDSETS),
-    (None, None, None, FIELDSETS_WITHOUT_RESUME_OR_COVER_LETTER)
-], ids=['with_resume', 'with_cover_letter_file', 'with_cover_letter_text', 'with_neither_resume_nor_cover_letter']
+@pytest.mark.parametrize('resume, interest_in_program, expected_fieldsets', [
+    (TEST_RESUME, None, ALL_FIELDSETS),
+    (None, TEST_INTEREST_IN_PROGRAM, ALL_FIELDSETS),
+    (None, None, FIELDSETS_WITHOUT_INTEREST_OR_RESUME)
+], ids=['with_resume', 'without_resume_and_interest', 'with_neither_resume_nor_interest']
 )
 @pytest.mark.django_db
 @mock.patch('openedx.adg.lms.applications.admin.UserApplicationADGAdmin._get_fieldset_for_scores')
-@mock.patch('openedx.adg.lms.applications.admin.UserApplicationADGAdmin._get_fieldset_for_resume_cover_letter')
+@mock.patch('openedx.adg.lms.applications.admin.UserApplicationADGAdmin._get_fieldset_for_resume_and_interest')
 @mock.patch('openedx.adg.lms.applications.admin.UserApplicationADGAdmin._get_applicant_info_fieldset')
 @mock.patch('openedx.adg.lms.applications.admin.UserApplicationADGAdmin._get_preliminary_info_fieldset')
 def test_get_fieldsets(
     mock_get_preliminary_info_fieldset,
     mock_get_applicant_info_fieldset,
-    mock_get_fieldset_for_resume_cover_letter,
+    mock_get_fieldset_for_resume,
```

Comment:
As we are mocking _get_fieldset_for_resume_and_interest hence we should rename mock_get_fieldset_for_resume to mock_get_fieldset_for_resume_and_interest to completely reflect the method.